packages/cardano-services/src/Program/programs/types.ts

```diff
@@ -49,13 +50,16 @@ export enum ProviderServerOptionDescriptions {
   UseTypeOrmStakePoolProvider = 'Enables the TypeORM Stake Pool Provider',
   UseBlockfrost = 'Enables Blockfrost cached data DB',
   UseQueue = 'Enables RabbitMQ',
-  PaginationPageSizeLimit = 'Pagination page size limit shared across all providers'
+  PaginationPageSizeLimit = 'Pagination page size limit shared across all providers',
+  HandlePolicyIds = 'Cardano Policy ID',
+  HandleProviderServerUrl = 'Server url for the Kora Labs handle provider'
```

Comment:
```suggestion
  HandleProviderServerUrl = 'URL for the Handle provider server'
```
We'll be refactoring to use https://github.com/input-output-hk/cardano-js-sdk/pull/750/ soon enough, so can use naming and descriptions with this in mind.

Reply:
aa8096909c8ee4b312f8d2d9395b28766ebd4ff2 should solve this